packages/storybook/src/documentation/pages/accessibility.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { XGrid } from '@material-ui/x-grid';
+
+<Meta title="1. Docs/12. Accessibility" component={XGrid} />
+
+# Accessibility
+
+Throughout our journey, accessibility has always been one of the core value of Material-UI components.
+Hence our data-grid has no less than 70 accessibility standards implemented in our first beta version.
+
+## Keyboard Navigation
+
+|                                     Keys | Description                                       |
+| ---------------------------------------: | ------------------------------------------------- |
```

Comment:
Not sure?
```suggestion
| ---------------------------------------: | :------------------------------------------------ |
```

Reply:
if you want...
By default, it's align left, but it does not seem to work anyway :/

Reply:
> By default, it's align left, but it does not seem to work anyway :/

By "it", do you mean mdx? GitHub aligns at the center by default, material-ui.com align center by default (to match GitHub)